app/controllers/admin/column_preferences_controller.rb

```diff
@@ -1,5 +1,5 @@
 module Admin
-  class ColumnPreferencesController < ResourceController
+  class ColumnPreferencesController < Admin::ResourceController
```

Comment:
I don't understand why this is necessary. Aren't we in the `Admin` namespace already? I tested it locally and the controller spec still passes without the `Admin::`. We do need `::Admin::` within the Spree controllers though because they are not in the same namespace.

Reply:
If we don't need it, it does look cleaner to not have it prefixed with an unnecessary `Admin::`. Now I'm curious why it wouldn't interpret it as being a subclass of the `Admin::Admin::ResourceController`. But in any case, if it's passing without it, I'd say remove it. 

Did you test just this controller, or all the similar controllers that have that prefix?

Reply:
I tested only that one controller. Names are looked up in several places. It starts in the current namespace and then goes up in the hierarchy until it finds something (or not). So if we reference `ResourceController` here, it will search for `Admin::ResourceController` because we are in the `Admin` module. If it wouldn't find that, it would look for a global `ResourceController`. This article is really good:
https://cirw.in/blog/constant-lookup.html

I'm having today and tomorrow off. Feel free to make the changes if you have time before Wednesday.

Reply:
I had a look at this and there isn't an isolated commit for this change. And there are existing references to `Admin::BaseController` already. So I'll move this to test-ready and open a new pull request to simplify this.